driver-core/src/main/com/mongodb/client/model/Aggregates.java

```diff
@@ -67,6 +69,21 @@ public static Bson addFields(final List<Field<?>> fields) {
         return new AddFieldsStage(fields);
     }
 
+    /**
+     * Creates an $addFields pipeline stage
+     *
+     * @param fields        the fields to add
+     * @return the $addFields pipeline stage
+     * @mongodb.driver.manual reference/operator/aggregation/addFields/ $addFields
+     * @mongodb.server.release 3.4
+     * @since 3.4
+     */
+    public static <TExpression> Bson addFields(final Map<String, TExpression> fields) {
```

Comment:
This seems more convenient than the existing overloads now that Java has `Map.of`